decidim-core/lib/decidim/core/engine.rb

```diff
@@ -32,6 +32,7 @@
 require "premailer/rails"
 require "nokogiri"
 require "geocoder"
+require "letter_opener_web"
```

Comment:
To clarify... if I put the require on `decidim/lib/decidim/dev.rb` (the original file) I cannot run the `generate_test_app` from an external engine.